src/client/common/components/base-network-view/base-network-view.js

```diff
@@ -163,16 +164,40 @@ class BaseNetworkView extends React.Component {
       }, [nodeSearchBarInput])
     ];
 
-
-
-
     const toolBar = [
       ...menuButtons,
       ...networkButtons,
       // ...(componentConfig.useLayoutDropdown ? layoutDropdown : []), // TODO re-add dropdown for edit
       ...(componentConfig.useSearchBar ? nodeSearchBar : [])
     ];
 
+    //display pathway and database names
+    const metadataTitles = h('h4',[
+      h('span', state.networkMetadata.name),
+      ' | ',
+      h('a', state.networkMetadata.datasource)
+    ]);
+
+    //display human icon indicating human genome, gene input bar, and submit button
+    const geneInputBar =  [
```

Comment:
To me, a `geneInputBar` feels like an item that is specific to a particular view. One idea is to let views control the entire `title-container` and default to the metadataTitles?